src/manage_sql_nvts.c

```diff
@@ -1289,7 +1291,14 @@ nvti_from_vt (entity_t vt)
                       "cvss_base_v2")
               == 0))
         {
+          gchar * cvss_base;
+
           nvti_add_tag (nvti, "cvss_base_vector", entity_text (severity));
+
+          cvss_base = g_strdup_printf ("%.1f", get_cvss_score_from_base_metrics
+                                           (entity_text (severity)));
```

Comment:
Looks like an indentation problem here.

Reply:
Fix is not what I expected :) but let's wait for the clang-format which will surely
change it anyway.